test/common/buffer/utility_test.cc

```diff
@@ -30,16 +30,19 @@ TEST(DataConstructorTest, FromCToCpp) {
 TEST(DataConstructorTest, FromCppToCEmpty) {
   OwnedImpl empty_data;
 
-  envoy_data c_data = Utility::toBridgeData(empty_data);
+  absl::optional<envoy_data> maybe_c_data = Utility::toBridgeData(empty_data);
```

Comment:
I'd suggest moving from `optional` to hard checks. If we see ourselves making a lot of these, we could introduce a c utility function `is_valid` (though I don't know whether this is warranted).